backend/src/journal/journal.service.ts

```diff
@@ -0,0 +1,101 @@
+import { Injectable } from '@nestjs/common';
+import { CreateJournalInput } from './dto/create-journal.input';
+import { UpdateJournalInput } from './dto/update-journal.input';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { Journal } from './entities/journal.entity';
+import { AuthService } from 'src/auth/auth.service';
+
+@Injectable()
+export class JournalService {
+  constructor(
+    private readonly prisma: PrismaService,
+    private readonly authService: AuthService,
+  ) {}
+
+  async getUserByAuthId(authId: string) {
```

Comment:
This doesn't belong in the journal service — since it's specifically for the user

Either remove it from here and just always use prisma method
Or create a user module, user service and add this method in there

Reply:
Done! I chose the first option, which is to create a user module and service, so I can use it in the future as well.